dbt_subprojects/dex/tests/generic/test_reasonable_usd_amount.sql

```diff
@@ -0,0 +1,121 @@
+{% test test_reasonable_usd_amount(model, column_name, max_value=1000000000, days_back=30) %}
```

Comment:
```suggestion
{% test test_acceptable_usd_amount(model, column_name, max_value=1000000000, days_back=30) %}
```

Reply:
minor name change, to match our existing test. should be able to search/replace in IDE to make the change quick 🤞